8-nodes-block-4-2/logs.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
You should put this playbook here https://github.com/ydb-platform/ydb-ansible/tree/main/playbooks
to avoid duplicate code and use it with any project.

Reply:
> @artgromov then why do we have setup_playbook.yaml copy-pasted to all examples?

I dont know, they seem excessive to me.

> How people are supposed to discover the playbooks in that folder?

Only by reading collection documentation, which is absent for now.

That is the usual way for ansible collections. By making people to copy-paste example playbooks you loosing maintainability for user-deployed systems. If we want to guarantee service continuity while using our playbooks, we should distribute them with ydb_platform collection and heavily use various asserts.

Reply:
Ok, so it sounds like a plan:

1. Here's a copy of this PR in that other repo: https://github.com/ydb-platform/ydb-ansible/pull/19
2. We'll need to update the documentation so people all users end up with Ansible 2.11+ (currently, I got 2.10 by following them on Ubuntu 22.04, and the steps below don't work)
3. Recommend people to use playbooks from our collection via https://docs.ansible.com/ansible/latest/collections_guide/collections_using_playbooks.html#using-a-playbook-from-a-collection (and move setup_playbook.yaml there as well)
4. Document other available playbooks.

Reply:
Copy of setup_playbook.yaml https://github.com/ydb-platform/ydb-ansible/pull/20

Reply:
Improvement of instructions to lower the chance of people accidentally ending up with an old Ansible version like I did: https://github.com/ydb-platform/ydb/pull/4394/files